engine/worker/builtin.go

```diff
@@ -99,6 +103,9 @@ func (w *currentWorker) runPlugin(ctx context.Context, a *sdk.Action, buildID in
 		}
 		for _, p := range params {
 			pluginArgs.Data[p.Name] = p.Value
+			if p.Type == sdk.SecretVariable {
```

Comment:
not sure of this type. I think there is a func sdk.NeedPlaceHolder or something...